bugsnag-plugin-android-ndk/src/main/jni/handlers/signal_handler.c

```diff
@@ -63,6 +63,85 @@ static const char bsg_native_signal_msgs[BSG_HANDLED_SIGNAL_COUNT + 1][60] = {
     "Segmentation violation (invalid memory reference)"};
 
+static const char bsg_native_signal_code_names[BSG_HANDLED_SIGNAL_COUNT +
+                                               1][BSG_SIGNAL_CODE_COUNT +
+                                                  1][72] = {
+    {"Illegal instruction, code 1 (ILLOPC)",
+     "Illegal instruction, code 2 (ILLOPN)",
+     "Illegal instruction, code 3 (ILLADR)",
+     "Illegal instruction, code 4 (ILLTRP)",
+     "Illegal instruction, code 5 (PRVOPC)",
+     "Illegal instruction, code 6 (PRVREG)",
+     "Illegal instruction, code 7 (COPROC)",
+     "Illegal instruction, code 8 (BADSTK)",
+     "Illegal instruction, code 9 (BADIADDR)",
+     "Illegal instruction, code 10 (BREAK)",
+     "Illegal instruction, code 11 (BNDMOD)"},
+    {"Trace/breakpoint trap, code 1 (TRAP_BRKPT)",
+     "Trace/breakpoint trap, code 2 (TRAP_TRACE)",
+     "Trace/breakpoint trap, code 3 (TRAP_BRANCH)",
+     "Trace/breakpoint trap, code 4 (TRAP_HWBKPT)",
+     "Trace/breakpoint trap, code 5 (TRAP_UNK)",
+     "Trace/breakpoint trap, code 6 (TRAP_PERF)"},
+    {0},
+    {"Bus error (bad memory access), code 1 (BUS_ADRALN)",
+     "Bus error (bad memory access), code 2 (BUS_ADRERR)",
+     "Bus error (bad memory access), code 3 (BUS_OBJERR)",
+     "Bus error (bad memory access), code 4 (BUS_MCEERR_AR)",
+     "Bus error (bad memory access), code 5 (BUS_MCEERR_AO)"},
+
+    {"Floating-point exception, code 1 (FPE_INTDIV)",
+     "Floating-point exception, code 2 (FPE_INTOVF)",
+     "Floating-point exception, code 3 (FPE_FLTDIV)",
+     "Floating-point exception, code 4 (FPE_FLTOVF)",
+     "Floating-point exception, code 5 (FPE_FLTUND)",
+     "Floating-point exception, code 6 (FPE_FLTRES)",
+     "Floating-point exception, code 7 (FPE_FLTINV)",
+     "Floating-point exception, code 8 (FPE_FLTSUB)",
+     "Floating-point exception, code 9 (__FPE_DECOVF)",
+     "Floating-point exception, code 10 (__FPE_DECDIV)",
+     "Floating-point exception, code 11 (__FPE_DECERR)",
+     "Floating-point exception, code 12 (__FPE_INVASC)",
+     "Floating-point exception, code 13 (__FPE_INVDEC)",
+     "Floating-point exception, code 14 (FPE_FLTUNK)",
+     "Floating-point exception, code 15 (FPE_CONDTRAP)"},
+    {"Segmentation violation (invalid memory reference), code 1 (SEGV_MAPERR)",
+     "Segmentation violation (invalid memory reference), code 2 (SEGV_ACCERR)",
+     "Segmentation violation (invalid memory reference), code 3 (SEGV_BNDERR)",
+     "Segmentation violation (invalid memory reference), code 4 (SEGV_PKUERR)",
+     "Segmentation violation (invalid memory reference), code 5 (SEGV_ACCADI)",
+     "Segmentation violation (invalid memory reference), code 6 (SEGV_ADIDERR)",
+     "Segmentation violation (invalid memory reference), code 7 (SEGV_ADIPERR)",
+     "Segmentation violation (invalid memory reference), code 8 (SEGV_MTEAERR)",
+     "Segmentation violation (invalid memory reference), code 9 "
+     "(SEGV_MTESERR)"},
```

Comment:
All of these code messages should have their code prefixes removed to align with the `ILL` messages. ie:
```
"Trace/breakpoint trap, code 1 (TRAP_BRKPT)"
```
should be
```
"Trace/breakpoint trap, code 1 (BRKPT)"
```
and so on.